### DISCUSSION TITLE
Printable documentation

### DISCUSSION THREAD

**Discussion by sadden3194** [netbox-community/netbox#16330] [General]
Good morning all!

I would like to know If there is a way to print the list of patchfeld front ports.

The "browser print function" shows a not optimzied version, with navigation tabs, ...

![image](https://github.com/netbox-community/netbox/assets/17011498/898b4495-f491-485a-96eb-b9ba7dcf8fa2)

**Comment by sadden3194** [comment]
no response??

**Comment by zmaster7** [comment]
Depending on how you'd like it formatted, you could always use [Export Templates](https://netboxlabs.com/docs/netbox/en/stable/customization/export-templates/). You would then have navigate to Devices -> Front Ports and filter the results before selecting Export. It's possible the Current View/All Data options are enough for your needs without having to resort to custom templates.

**Comment by sadden3194** [comment]
"You would then have navigate to Devices -> Front Ports and filter the results before selecting Export."
There is no optioon to export??

**Reply by sadden3194** [reply]
Ah okay I found it.
Is there a way to save the different "filter" for export?

**Reply by zmaster7** [reply]
Any applied filter can be saved with the "save" option next to the applied filters on the "results" tab. To re-apply saved filters you would switch over to "filter" and select it from the "saved filter" dropdown.

![image](https://github.com/netbox-community/netbox/assets/46574676/29fb1cec-28e7-4e99-ba33-c179150f5c6e)

![image](https://github.com/netbox-community/netbox/assets/46574676/7020a8b3-1892-40b9-9dde-b58a428a5165)

![image](https://github.com/netbox-community/netbox/assets/46574676/b5c697cf-5c53-4240-a124-838dd7c08e9a)

**Comment by Kage1** [comment]
Sounds like you've got it sorted, but as a quick and dirty workaround I highlight all of the headers and text and then paste them into an Excel document. It mostly gets the formatting correct.
